wdl/structs/hotspot1.wdl

```diff
@@ -0,0 +1,13 @@
+version 1.0
+
+
+struct HotSpot1Index {
```

Comment:
I had a long think about naming here. In my humble opinion `HotSpot1Reference` would be more spot on. That said, if you think the search-and-replace is too painful, then don't do it.

Reply:
Good suggestion.

Reply:
Fixed.